addOns/automation/src/main/java/org/zaproxy/addon/automation/ExtensionAutomation.java

```diff
@@ -678,7 +677,6 @@ private void runPlanCommandLine(String source) {
 
     private static URI createUri(String source) {
         try {
-            new URL(source).toURI();
```

Comment:
This should be kept but changed to not use deprecated constructor, it validates different things.

Reply:
Okay